### PR TITLE
Misc. invisible changes to links and metadata

### PR DESCRIPTION
**Pull request by YoshiRulz**
Broadly SEO-related, though I won't claim this kind of change will improve, or have any impact on, the site's ranking in Google results.

edit: I get the feeling this is more likely to get merged if it's split into multiple PRs, but I have several open already, so I'll do that later.

### CODE REVIEW

File: TASVideos.WikiEngine/MakeBracketed.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ public static partial class Builtins
 	/// <summary>
 	/// Turns text inside [square brackets] into the appropriate thing, usually module or link.  Does not handle [if:].
 	/// </summary>
-	public static IEnumerable<INode> MakeBracketed(int charStart, int charEnd, string text)
+	public static IEnumerable<INode> MakeBracketed(int charStart, int charEnd, bool isUGC, string text)
```
Comment (adelikat) [inline TASVideos.WikiEngine/MakeBracketed.cs:15]: What is UGC?

Comment (YoshiRulz) [reply TASVideos.WikiEngine/MakeBracketed.cs:15]: User-generated content https://developers.google.com/search/docs/crawling-indexing/qualify-outbound-links#ugc

Comment (adelikat) [reply TASVideos.WikiEngine/MakeBracketed.cs:15]: I'm struggling to understand the code here.  How does it determine which Wiki pages are UGC and which aren't?  Shouldn't we consider ALL wiki pages as UGC? It's a wiki.

Comment (YoshiRulz) [reply TASVideos.WikiEngine/MakeBracketed.cs:15]: For wiki-lang content, I've done `IsUGC => wikiPage.IsHomePageOrSubmissionNotes();`

**Comment by adelikat** [comment]
Closing for comments above

**Pull request closed without merging**